include/jemalloc/internal/ph.h

```diff
@@ -127,6 +127,7 @@ phn_merge_ordered(void *phn0, void *phn1, size_t offset,
 	phn0child = phn_lchild_get(phn0, offset);
 	phn_next_set(phn1, phn0child, offset);
 	if (phn0child != NULL) {
+		/* NOLINTNEXTLINE(readability-suspicious-call-argument) */
```

Comment:
Question: if we put the NOLINT into phn_prev_ / _next_set, would that work too?

Reply:
I believe it has to be at the callsite, but even if not, I would advise leaving it as-is, because while these are false-positives, I think the lint has value in general. For example if you had code like:
```c
void *root = ...;
void *prev = ...;
...
phn_prev_set(prev, root, OFFSET);
```

Then the lint would save you from a mistake where you accidentally flipped the order of the arguments like that.